fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/impl/networking/NetworkingImpl.java

```diff
@@ -38,4 +51,12 @@ public final class NetworkingImpl {
 	public static boolean isReservedCommonChannel(Identifier channelName) {
 		return channelName.equals(REGISTER_CHANNEL) || channelName.equals(UNREGISTER_CHANNEL);
 	}
+
+	static {
+		if (FORCE_PACKET_SERIALIZATION) {
+			LOGGER.warn("Force Packet Serialization is enabled");
+			if (FabricLoader.getInstance().isDevelopmentEnvironment()) LOGGER.warn("This is the default on dev env");
+			LOGGER.warn("Disable it with -Dfabric-api.networking.force-packet-serialization=false");
+		}
```

Comment:
Im not sure these need to be warning levels, or logged at all in development tbh.

Reply:
How does one know to disable the serialization then? I guess most devs won't need to be concerned about this but still.

Reply:
Why would one want to disable it in the first place? ... ;)

Reply:
To make sure it doesn't bug on singleplayer, because of improper copy of an object?